packages/rancher-monitoring/rancher-monitoring.patch

```diff
@@ -2359,15 +2350,15 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-monitoring/charts-original/val
      dashboards:
        enabled: true
        label: grafana_dashboard
-+      searchNamespace: grafana-dashboards
++      searchNamespace: cattle-dashboards
  
        ## Annotations for Grafana dashboard configmaps
        ##
 @@ -548,6 +909,7 @@
        ## ref: https://git.io/fjaBS
        createPrometheusReplicasDatasources: false
        label: grafana_datasource
-+      searchNamespace: grafana-datasources
++      # searchNamespace: "cattle-monitoring-system"
```

Comment:
Why is this updated and commented out?

Reply:
Based on https://github.com/rancher/charts/pull/648/files#diff-ed5b5b87cf149adc6a6408bf81993efdR848, we'll either take the value provided here or default to `cattle-monitoring-system`.

I could omit the comment here, but I thought it would just be helpful for a user to glance at to know where they should expect their datasources to be.

Reply:
If you want to leave the comment can you expand with what you just said? Otherwise I think it's missing context

Reply:
on second thought, I think it's better to omit this; a user can always find this information out by looking at the relevant README on the Grafana helm chart, so it doesn't make sense for us to expose this here as a commented out option.